wls-gui-wahllokalsystem/src/components/wahlvorbereitung/BaseWahlschliessungCard.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-card>
+    <v-card-title>Wahlscheine und Uhrzeit Wahlende</v-card-title>
+    <v-card-text class="pb-0">
+      Bitte geben Sie hier die Uhrzeit ein, zu der die Wahl für geschlossen
+      erklärt wurde.
+      <v-form v-model="schliessungsuhrzeitValidForm">
+        <base-time-input v-model="schliessungsuhrzeit" />
+      </v-form>
+    </v-card-text>
+    <v-card-actions>
+      <base-button-save
+        color="primary"
+        :disabled="isSaveButtonDisabled"
+        @click="onSaveSchliessungsuhrzeitClicked"
+      />
+    </v-card-actions>
+  </v-card>
+</template>
+
+<script setup lang="ts">
+import type { Ref } from "vue";
+
+import { computed, ref } from "vue";
+import {
+  VCard,
+  VCardActions,
+  VCardText,
+  VCardTitle,
+  VForm,
+} from "vuetify/components";
+
+import BaseButtonSave from "@/components/common/buttons/BaseButtonSave.vue";
+import BaseTimeInput from "@/components/common/inputs/BaseTimeInput.vue";
+import { useWahlbezirkStore } from "@/stores/wahlbezirkStore.ts";
+
+const schliessungsuhrzeitValidForm: Ref<null | boolean> = ref(null);
+const schliessungsuhrzeit: Ref<Date | undefined> = ref(undefined);
+
+const wahlbezirkStore = useWahlbezirkStore();
+const isSaveButtonDisabled = computed(
```

Comment:
steigert aus meiner Sicht etwas die Lesbarkeit, da die beiden Konstanten für mich keinen logischen Block bilden.

```suggestion
const wahlbezirkStore = useWahlbezirkStore();

const isSaveButtonDisabled = computed(
```

Reply:
done [cea8f3d](https://github.com/it-at-m/Wahllokalsystem/pull/1241/commits/cea8f3daef6569b5141b5f7c6592330a64a702d6)